### PR TITLE
Update tutorial

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -21,12 +21,12 @@ Install from PyPI::
 
   pip install resdk
 
-.. TODO: Remove this warning when not needed anymore:
+.. TODO: Remove this warning when not needed anymore: Miha, Domen is this still relevant?
```
Comment (JureZmrzlikar) [inline docs/index.rst:24]: @mstajdohar @dblenkus

Comment (mstajdohar) [reply docs/index.rst:24]: @dblenkus ?

**Review by mstajdohar — APPROVED** [review]
Looks good. One minor comment, capitalize both words in Genialis Server.

**Comment by mstajdohar** [comment]
@jinlajf, you asked for this a while ago.

**Comment by jinlajf** [comment]
Thanks. :) 

I asked Mia to review the tutorials and comment. I think we could ask the same from Lara and Robert, too. @mstajdohar

**Comment by JureZmrzlikar** [comment]
> capitalize both words in Genialis Server.

Done.

**Comment by mstajdohar** [comment]
What's up with tests?

**Comment by JureZmrzlikar** [comment]
> What's up with tests?

Black version is not pinned, ergo black is now 20.x instead of 19.x and we have:
  - A dependency hell (one dependency requires click>7.x other click <=6.x , )...
  - New version of black enforces different formatting, ergo, all the code is now "wrongly" formatted.

Fun, fun, fun...

**Comment by mstajdohar** [comment]
Is it ok to loosen the dependencies?

**Comment by JureZmrzlikar** [comment]
> Is it ok to loosen the dependencies?

Not *per se*. I've fixed that. I've only done minimal changes to make stuff work.

**Comment by mstajdohar** [comment]
`pip check` fails now :)

**Comment by mstajdohar** [comment]
Hooray, thanks!

**Pull request merged**